capstone/capweb/urls.py

```diff
@@ -48,6 +48,7 @@
     ### downloads ###
     path('download', views.download_files, name='download-files'),
     re_path(r'download/(?P<filepath>.*)', views.download_files, name='download-files-in-path'),
+    path('download_contents', views.download_contents_file, name='download-contents'),
```

Comment:
Let's put this above the previous line, with the path "download/manifest.csv", so it works more like a real file.